test/Microsoft.ML.StaticPipelineTesting/StaticPipeTests.cs

```diff
@@ -431,5 +432,39 @@ public void Tokenize()
             Assert.True(type.IsVector && !type.IsKnownSizeVector && type.ItemType.IsKey);
             Assert.True(type.ItemType.AsKey.RawKind == DataKind.U2);
         }
+
+
+        [Fact]
+        public void LpGcNormAndWhitening()
+        {
+            var env = new ConsoleEnvironment(seed: 0);
+            var dataPath = GetDataPath("generated_regression_dataset.csv");
+            var dataSource = new MultiFileSource(dataPath);
+
+            var reader = TextLoader.CreateReader(env,
+                c => (label: c.LoadFloat(11), features: c.LoadFloat(0, 10)),
```

Comment:
>11 [](start = 41, length = 2)

these are 0-based, are you sure?

Reply:
These is i think.

features are from 0-10 and label is 11. Do you mean something else?

---
In reply to: [219263470](https://github.com/dotnet/machinelearning/pull/961#discussion_r219263470) [](ancestors = 219263470)

Reply:
Yea, it's my mistake

---
In reply to: [219289826](https://github.com/dotnet/machinelearning/pull/961#discussion_r219289826) [](ancestors = 219289826,219263470)